### ISSUE TITLE
Use a separate directory for aux/fdb_latexmk/fls/log/synctex.gz files?

### ISSUE THREAD

**Issue by sorsasampo** [lervag/vimtex#1297]
Using vimtex with latexmk seems to create `.{aux,fdb_latexmk,fls,log,synctext.gz}` files in the same directory as the .tex file.

Is there a way to move these files into a separate temporary directory?

**Comment by sorsasampo** [comment]
Another option seems to be to create `~/.latexmkrc` and set `$out_dir` in it.

**Comment by lervag** [comment]
Yes (although it is implied by the docs that these things may also be set in the `.latexmkrc` file).
